src/info/utils/mod.rs

```diff
@@ -24,20 +24,26 @@ where
 }
 
 fn to_human_time(time: Time) -> String {
-    let since_epoch_duration = SystemTime::now()
-        .duration_since(SystemTime::UNIX_EPOCH)
-        .unwrap();
+    let duration = match diff_gix_time(SystemTime::now(), time) {
+        Ok(d) => d,
+        Err(s) => return s,
+    };
+    let ht = HumanTime::from(-(duration.as_secs() as i64));
+    ht.to_string()
+}
 
-    let ts = Duration::from_secs(match time.seconds.try_into() {
+/// Gets the duration between `now` and `time`. Returns `Err` if this cannot be calculated.
+fn diff_gix_time(now: SystemTime, time: Time) -> Result<Duration, String> {
+    let since_epoch_duration = now.duration_since(SystemTime::UNIX_EPOCH).unwrap();
+    let ts = Duration::from_secs(match (time.seconds + i64::from(time.offset)).try_into() {
         Ok(s) => s,
-        Err(_) => return "<before UNIX epoch>".into(),
+        Err(_) => return Err("<before UNIX epoch>".into()),
```

Comment:
`gitoxide` supports times before unix epoch, but of course, there are dates outside of the valid range of `i64`.